src/main/api/2/studio-api-2.yaml

```diff
@@ -1530,6 +1530,24 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /cluster/setPrimary:
```

Comment:
Change to require the ManagementToken as we did for the Monitoring APIs

Reply:
Switched to ManagementToken.